zyte_common_items/components/__init__.py

```diff
@@ -21,4 +21,5 @@
 from .ratings import AggregateRating, StarRating
 from .real_estate import RealEstateArea
 from .request import Header, ProbabilityRequest, Request
+from .serp import SerpOrganicResult
```

Comment:
Currently for other listing items list elements are not in the components module, they're in items directly - see e.g. product list or article list.